cube/CMakeLists.txt

```diff
@@ -174,6 +174,12 @@ elseif(NOT WIN32)
                        cube.vert.inc
                        cube.frag.inc)
         target_link_libraries(vkcube Vulkan::Vulkan)
+        if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
```

Comment:
Is this CMAKE_SYSTEM_NAME check actually required?  By the outer if-else tree, it's already !APPLE and !WIN32.

Reply:
Ok, removed